pylinac/planar_imaging.py

```diff
@@ -738,7 +738,7 @@ def _phantom_angle_calc(self) -> float:
 
         start_angle_deg = self._determine_start_angle_for_circle_profile()
         circle = self._circle_profile_for_phantom_angle(start_angle_deg)
-        peak_idx = circle.find_fwxm_peaks(threshold=0.6, max_number=1)[0]
+        peak_idx = circle.find_fwxm_peaks(threshold=0.6, x=80, max_number=1)[0]
```

Comment:
This x=80 is failing a number of tests. Why are you choosing 80?

Reply:
No good reason...I think that I was somewhat confused and was experimenting with various parameters. (I'm not proud :frowning_face:  )